tmt/steps/provision/local.py

```diff
@@ -13,6 +13,9 @@
 from tmt.utils.hints import get_hint
 from tmt.utils.wait import Waiting
 
+#: The default helper scripts destination directory
+DEFAULT_HELPER_SCRIPTS_DEST_DIR = Path("/var/lib/tmt/helper/scripts")
```

Comment:
@mcasquer I would move this default to the other ones, so they are on the same place maybe? and just call it `DEFAULT_SCRIPTS_DEST_DIR_LOCAL` ?

see tmt/steps/scripts.py

Reply:
So finally do we rename the constant or as it's going to be in local.py the naming doesn't longer matter

Reply:
Renaming can't hurt, but I would keep it in `local.py`.